### PR TITLE
Statewise charts

### PR DESCRIPTION
**Pull request by yogigachinmath**
Preview:

Screen Shot 2020-04-05 at 3 05 58 AM

### CODE REVIEW

**Comment by thahir007cp** [comment]
Scale modes not working. Either Deactivate "uniform" switch like the other done or should able to switch to a uniform scale
Screenshot 2020-04-04 at 11 37 39 PM

File: src/components/home.js [hunk 1]
```diff
@@ -114,6 +114,7 @@ function Home(props) {
                 <div
                   className={`tab ${graphOption === 1 ? 'focused' : ''}`}
                   onClick={() => {
+                    console.log(graphOption);
```
Comment (bdpiprava) [inline src/components/home.js]: Please can you remove console log

Comment (yogigachinmath) [reply src/components/home.js]: Hi,
I have removed console, if you want to pull, i will merge with my master you can pull from there.

**Comment by yogigachinmath** [comment]
Can any one check this please, or else please close the pull request

**Comment by JunaidBabu** [comment]
This is covered in deep dive section.

**Pull request closed without merging**